### PR TITLE
Update opportunity types

### PR DESCRIPTION
**Pull request by hskang9**
resolves #5557

### CODE REVIEW

File: packages/apps-config/src/api/spec/opportunity.ts [hunk 1]
```diff
@@ -38,7 +38,15 @@ const definitions: OverrideBundleDefinition = {
           ]
         },
         AmountOf: 'Amount',
-        Amount: 'i128'
+        Amount: 'i128',
```
Comment (jacogr) [inline packages/apps-config/src/api/spec/opportunity.ts:41]: Not sure I see `CurrencyIdOf` (mentioned in https://github.com/polkadot-js/apps/issues/5571) - unless removed now on the chain itself.

Comment (jacogr) [reply packages/apps-config/src/api/spec/opportunity.ts:41]: Yes, that is exactly it - the type doesn't appear in the type definitions, hence the question.

Comment (hskang9) [reply packages/apps-config/src/api/spec/opportunity.ts:41]: do you mean the type definition [here](https://github.com/polkadot-js/apps/blob/fad1f3da1076e085cfaa35133adf573114912f0f/packages/apps-config/src/api/spec/opportunity.ts#L22) or somewhere else? I think CurrencyIdOf in our runtime is from the code [here](https://github.com/digitalnativeinc/standard-substrate/blob/4a2f50689975d2a723b12c864e3f8233ed141e55/orml/currencies/src/lib.rs#L88)

Comment (hskang9) [reply packages/apps-config/src/api/spec/opportunity.ts:41]: > Not sure I see `CurrencyIdOf` (mentioned in #5571) - unless removed now on the chain itself.

This PR includes `CurrencyIdOf`, and it passed `yarn ci:chainTypes` command.

Comment (hskang9) [reply packages/apps-config/src/api/spec/opportunity.ts:41]: > Not sure I see `CurrencyIdOf` (mentioned in #5571) - unless removed now on the chain itself.

Ah...now I understand. I set the testing unreachable to check if there are outdated types.

File: packages/apps-config/src/endpoints/testing.ts [hunk 2]
```diff
@@ -263,6 +263,7 @@ export function createTesting (t: TFunction, firstOnly?: boolean): LinkOption[]
     },
     {
       info: 'opportunity',
+      isUnreachable: true,
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/testing.ts:266]:
```suggestion
```

Should be fine without.

**Review by jacogr — APPROVED** [review]
Thank you. Should fix https://github.com/polkadot-js/apps/issues/5577

**Pull request merged**